cypress/support/commands.js

```diff
@@ -1,22 +1,77 @@
 // Copyright (c) Cosmo Tech.
 // Licensed under the MIT license.
-import { SELECTORS } from '../constants/IdConstants';
-import {
-  PAGE_NAME,
-  URL_ROOT
-} from '../constants/TestConstants.js';
-
-const urlRegexWithAnySuffix = new RegExp(`^${URL_ROOT}/.*${PAGE_NAME.SCENARIOS}/.*`);
+import { SELECTORS } from '../commons/IdConstants.js';
+import { URL_REGEX } from '../commons/TestConstants.js';
 
 // For this moment, login is handle by clicking on Microsoft Login button
 Cypress.Commands.add('login', () => {
-  cy.intercept('GET', urlRegexWithAnySuffix).as('requestUpdateCurrentScenario1');
+  // Stub PowerBi request
+  cy.intercept('GET', 'http://localhost:3000/api/get-embed-info', {
```

Comment:
`http://localhost:3000/api` should be extract in constant in general config file